packages/kujua-sms/views/lib/smsparser.js

```diff
@@ -1,4 +1,5 @@
 var utils = require('kujua-utils'),
+    standardiseDigits = require('eurodig'),
```

Comment:
Basically to use a require within couchdb I think the code needs to reside under `views/lib` on the ddoc.  That's why the `smsparser` module has that weird path inside the kanso package dir.